iina.xcodeproj/project.pbxproj

```diff
@@ -1989,6 +1994,15 @@
 /* End PBXSourcesBuildPhase section */
 
 /* Begin PBXVariantGroup section */
+		134DAFDA207C288200F116BD /* SubChooseViewController.xib */ = {
```

Comment:
It seems like you accidentally removed the reference of this file, and re-added it manually. The reference is now changed, and original translation are missing because of it. Try to revert it as well.